cmake/kokkos_compiler_id.cmake

```diff
@@ -33,6 +33,21 @@ IF(INTERNAL_HAVE_COMPILER_NVCC)
   STRING(SUBSTRING ${TEMP_CXX_COMPILER_VERSION} 1 -1 TEMP_CXX_COMPILER_VERSION)
   SET(KOKKOS_CXX_COMPILER_VERSION ${TEMP_CXX_COMPILER_VERSION} CACHE STRING INTERNAL FORCE)
   MESSAGE(STATUS "Compiler Version: ${KOKKOS_CXX_COMPILER_VERSION}")
+
```

Comment:
Is there a reason we can't just set `KOKKOS_CXX_HOST_COMPILER_ID` to `CMAKE_CXX_COMPILER_ID`? These are same already, unless we override it to NVIDIA or HIP.

Reply:
Yes, that works and is simpler. :slightly_smiling_face: